tmt/base.py

```diff
@@ -366,14 +366,13 @@ def lint(self):
             valid = False
 
         # Check test, path and summary
-        echo(verdict(self.test is not None, 'test script must be defined'))
-        echo(verdict(stripped_path.startswith('/'),
-                     'directory path must be absolute'))
-        echo(verdict(os.path.exists(test_path), 'directory path must exist'))
+        verdict(bool(self.test), 'test script must be defined')
```

Comment:
IMHO we can remove line 360 `valid = self.test is not None` and 365,366 ` if not stripped_path.s..` and make instead
```python
valid = verdict(bool(self.test), 'test script must be defined')
valid = valid and verdict(stripped_path.startswith('/'), 'directory path must be absolute')
... etc...
```

Reply:
oh right, thx, should be resovled now